.github/workflows/redis-cluster.yml

```diff
@@ -0,0 +1,56 @@
+name: Test Redis Cluster
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+# Cancel in progress workflows on pull_requests.
+# https://docs.github.com/en/actions/using-jobs/using-concurrency#example-using-a-fallback-value
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  redis_cluster:
```

Comment:
let's add this directly to `backend.yml` so we can make it required

Reply:
Ah true!